src/computed/channel.js

```diff
@@ -35,6 +38,15 @@ const ComputedChannel = store => {
     get channelBalanceOpenLabel() {
       const { channels, settings } = store;
       const sum = (channels || [])
+        .filter(c => c.active)
+        .map(c => Number(c.localBalance))
+        .reduce((a, b) => a + b, 0);
+      return toAmountLabel(sum, settings);
+    },
+    get channelBalanceInactiveLabel() {
+      const { channels, settings } = store;
+      const sum = (channels || [])
+        .filter(c => !c.active)
```

Comment:
@valentinewallace I also added `filter(c => !c.active)` here instead of the ternary expression. Seems a bit cleaner.